eg.overrides.js

```diff
@@ -0,0 +1,30 @@
+module.exports = {
+    'check window fully with html scrollRootElement after scroll when fail to scroll with css stitching': {features: ['useLocalDriver'], useLocalDriver: true},
+    'check window fully with html scrollRootElement after scroll when fail to scroll with scroll stitching': {features: ['useLocalDriver'], useLocalDriver: true},
+    'check window fully with custom scroll root with css stitching': {features: ['useLocalDriver'], useLocalDriver: true},
+    'check window fully with custom scroll root with scroll stitching': {features: ['useLocalDriver'], useLocalDriver: true},
+    'check frame in frame fully with css stitching': {features: ['useLocalDriver'], useLocalDriver: true},
+    'check frame in frame fully with scroll stitching': {features: ['useLocalDriver'], useLocalDriver: true},
+    'check window fully and frame in frame fully with css stitching': {features: ['useLocalDriver'], useLocalDriver: true},
+    'check window fully and frame in frame fully with scroll stitching': {features: ['useLocalDriver'], useLocalDriver: true},
+    'check frame after manual switch to frame with css stitching classic': {features: ['useLocalDriver'], useLocalDriver: true},
+    'check frame after manual switch to frame with scroll stitching classic': {features: ['useLocalDriver'], useLocalDriver: true},
+    'check region by selector in frame in frame fully with css stitching': {features: ['useLocalDriver'], useLocalDriver: true},
+    'check region by selector in frame in frame fully with scroll stitching': {features: ['useLocalDriver'], useLocalDriver: true},
+    'check region by selector in overflowed frame with css stitching': {features: ['useLocalDriver'], useLocalDriver: true},
+    'check region by selector in overflowed frame with scroll stitching': {features: ['useLocalDriver'], useLocalDriver: true},
```

Comment:
I see that flag is called `useLocalDriver` it means that without it flag it will running not locally. Am I right?
And why do you need a `features` then? Also please follow a semantic for `features`, there should be names of the features that might be not supported by all of the sdks (e.g. native-selectors or webdriver)

Reply:
changed flag to the `executionGrid:true`, `executionGrid:false`